Javadoc Warnings in the Gradle plugin

### Description

While building Quarkus (using `./mvnw -B clean install -DskipTests -DskipITs -DskipDocs -Prelocations`), I get the following warnings while building the Gradle plugin: 

```bash
[INFO] --- exec-maven-plugin:3.0.0:exec (gradle) @ io.quarkus.gradle.plugin ---
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 
> Task :gradle-application-plugin:clean UP-TO-DATE
> Task :gradle-model:compileJava UP-TO-DATE
> Task :gradle-application-plugin:compileJava
> Task :gradle-application-plugin:pluginDescriptors
> Task :gradle-application-plugin:processResources
> Task :gradle-application-plugin:classes
> Task :gradle-application-plugin:jar
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java:141: warning: no @param for fileCollection
    public static File getLastFile(FileCollection fileCollection) {
> Task :gradle-application-plugin:javadoc
                       ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java:141: warning: no @return
    public static File getLastFile(FileCollection fileCollection) {
                       ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:131: warning: no @return
    public Configuration getQuarkusDevConfiguration() {
                         ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:141: warning: no @return
    public FileCollection getSources() {
                          ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:151: warning: no @return
    public FileCollection getCompilationOutput() {
                          ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:162: warning: no @return
    public Property<File> getWorkingDirectory() {
                          ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:170: warning: no @param for workingDir
    public void setWorkingDir(String workingDir) {
                ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:185: warning: no @return
    public boolean isPreventnoverify() {
                   ^
/home/runner/work/quarkus-prettytime/quarkus-prettytime/quarkus/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java:197: warning: no @param for preventNoVerify
    public void setPreventnoverify(boolean preventNoVerify) {
                ^
9 warnings
```

### Implementation ideas

Fix the Javadoc warnings

--- a/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java
+++ b/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/extension/QuarkusPluginExtension.java
@@ -204,6 +204,9 @@ public abstract class QuarkusPluginExtension extends AbstractQuarkusExtension {
 
     /**
      * Returns the last file from the specified {@link FileCollection}.
+     *
+     * @param fileCollection the collection of files present in the directory
+     * @return result returns the last file
      */
     public static File getLastFile(FileCollection fileCollection) {
         File result = null;

--- a/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java
+++ b/devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java
@@ -142,6 +142,8 @@ public abstract class QuarkusDev extends QuarkusTask {
     /**
      * The dependency Configuration associated with this task. Used
      * for up-to-date checks
+     *
+     * @return quarkusDevConfiguration returns the configuration
      */
     @SuppressWarnings("unused")
     @CompileClasspath
@@ -151,6 +153,8 @@ public abstract class QuarkusDev extends QuarkusTask {
 
     /**
      * The JVM sources (Java, Kotlin, ..) for the project
+     *
+     * @return the FileCollection of all java source files present in the source directories
      */
     @Optional
     @InputFiles
@@ -161,6 +165,8 @@ public abstract class QuarkusDev extends QuarkusTask {
 
     /**
      * The JVM classes directory (compilation output)
+     *
+     * @return the FileCollection of all java source files present in the source directories
      */
     @Optional
     @InputFiles
@@ -174,6 +180,8 @@ public abstract class QuarkusDev extends QuarkusTask {
      *
      * Defaults to the main source set's classes directory. If there are
      * multiple, one is picked at random (see {@link QuarkusPluginExtension#getLastFile}).
+     *
+     * @return workingDirectory
      */
     @Input
     public Property<File> getWorkingDirectory() {
@@ -204,6 +212,7 @@ public abstract class QuarkusDev extends QuarkusTask {
     }
 
     /**
+     * @return boolean value of getPreventNoVerify()
      * @deprecated see {@link #getPreventNoVerify()}
      */
     @SuppressWarnings("SpellCheckingInspection")

